project/.travis.yml

```diff
@@ -0,0 +1,65 @@
+# Continuous Integration (CI) is the practice, in software
```

Comment:
To the best of my knowledge it was added during the Platformio Init. You could try deleting it and seeing if it has any effect on the system, as it is possible that it was added from some other package that I was using.